payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/CardDetailsController.kt

```diff
@@ -6,17 +6,25 @@ import java.util.UUID
 
 internal class CardDetailsController constructor(
     context: Context,
-    initialValues: Map<IdentifierSpec, String?>
+    initialValues: Map<IdentifierSpec, String?>,
+    viewOnlyFields: Set<IdentifierSpec>
```

Comment:
It seems the only valid value here is IdentifierSpec.CardNumber, as IdentifierSpec.Cvc will have no effect.   I would rename this cardNumberReadOnly to make it clear.

Reply:
Correct. I was making it generic all the way down, but changed here.